src/components/ABVCalculator.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <form id="abvCalc" class="max-w-48 flex flex-col">
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit ABV in %
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="spiritABV"
+        id="ABV"
+        placeholder="37.5"
+      />
+    </label>
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit quantity in ml
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="spiritQuantity"
+        id="alcoholQuantity"
+        placeholder="50"
+      />
+    </label>
+    <label class="justify-center w-full flex flex-col my-4"
+      >Mixer quantity in ml
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="mixerQuantity"
+        id="mixerQuantity"
+        placeholder="330"
+      />
+    </label>
+  </form>
+
+  <div class="results">
+    <p>Your drink is <span :class="computedABVColourClasses" class="font-bold">{{ `${calculateTotalDrinkABV}% ABV` }}</span>
+    </p>
+    <p>
+      There is <span class="font-bold" :class="computeTotalAlcoholColourClasses">{{ calculateTotalAlcoholQuantity }}ml</span> of alcohol in your drink
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      spiritABV: 37.5,
+      spiritQuantity: 50,
+      mixerQuantity: 330,
+    };
+  },
+  computed: {
+    abvToDecimal() {
+      return this.spiritABV / 100;
+    },
+    calculateTotalAlcoholQuantity() {
+      return this.spiritQuantity * this.abvToDecimal;
+    },
+    totalDrinkQuantity() {
+      return this.mixerQuantity + this.spiritQuantity;
+    },
+    calculateTotalDrinkABV() {
+      if (this.totalDrinkQuantity == 0 ){return 0}

```

Comment:
Brackets aren't needed and add to reading complexity
should be 
```
if (this.totalDrinkQuantity === 0 ) return 0;
```

Also `==` should be `===`

Reply:
Shouldn't this also be `this.totalDrinkQuantity()`?

Reply:
The brackets were added by my default Prettier which runs ESLint, but all the formatting errors like spacing, weird line breaks and shit are resolved if I format with Volar so I will try and set this as my default for .vue files
![image](https://user-images.githubusercontent.com/20053429/172496418-f25b93b6-4f2d-4357-9f19-dd0ee1e97d87.png)